master/Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM erlang:21
+FROM erlang:23
```

Comment:
The changelog for 1.11 says it only supports OTP 23+: https://github.com/elixir-lang/elixir/blob/82c4c3616b1dd79c4304050a687ba4eab609275b/CHANGELOG.md